subgraph/abis/PublicLock.json

```diff
@@ -3223,7 +3223,7 @@
       {
         "indexed": false,
         "internalType": "uint256",
-        "name": "maxKeysPerAddress",
+        "name": "maxKeysPerAcccount",
```

Comment:
We don't need this here, do we?

Reply:
this is automatically changed by building smart contract.

Reply:
resolved again